docs/_sources/_static/custom.css

```diff
@@ -1,61 +1,4 @@
-:root {
-  --bg-dark: #11303D;
-  --bg-light: #F0F0F0;
-}
-
-html {
-  background-color: var(--bg-dark, #11303D);
-}
-
-body {
-  background-color: var(--bg-light, #F0F0F0);
-}
-
-div.body {
-  max-width: 100%;
-}
-div.bodywrapper p {
-  margin-block: 0;
-}
-div.bodywrapper section {
-  background-color: white;
-  padding: 0 20px 30px 20px;
-}
-div.bodywrapper section > p {
-  padding-bottom: 1ex;
-}
-div.bodywrapper section h2 {
-  margin-block-start: 0;
-  padding-block-start: 0.83em;
-}
-div.citation p {
-  display: inline;
-}
-div.flowchart-container {
-  max-width: 1200px;
-}
-div.footer {
-  background-color: var(--bg-dark, #11303D);
-}
-div.sphinxsidebar {
-  background-color: var(--bg-light, #F0F0F0);
-}
-
-object {
-  max-width: 100%;
-}
-
-.bibtex .brackets::before {
-  content: "[";
-}
-.bibtex .brackets::after {
-  content: "]"
-}
-
-.bibtex.label {
-  display: grid;
-}
-
-section[id^="Tutorial:"] p img {
-  max-width: 80%;
```

Comment:
I'm not sure about the rest of the custom CSS with the changed theme, but I think we'll still need https://github.com/FCP-INDI/fcp-indi.github.io/blob/0a067870948763247e7311cf2a1b83e0ccd27053/docs/_sources/_static/custom.css#L31-L33 and these lines.

We can see how [fcp-indi.github.io/docs/nightly/user/tutorials/observed_usage](https://fcp-indi.github.io/docs/nightly/user/tutorials/observed_usage) looks with and without https://github.com/FCP-INDI/fcp-indi.github.io/blob/0a067870948763247e7311cf2a1b83e0ccd27053/docs/_sources/_static/custom.css#L59-L61 and 
![this part](https://user-images.githubusercontent.com/5974438/217275304-f8aebc80-c672-45a9-9b96-e28f32b8a879.png) and 
![this part](https://user-images.githubusercontent.com/5974438/217275387-c760f48d-aae9-40d3-9e48-11e8c52b0e84.png) on [fcp-indi.github.io/docs/nightly/user/tse](https://fcp-indi.github.io/docs/nightly/user/tse) should be a good example of https://github.com/FCP-INDI/fcp-indi.github.io/blob/0a067870948763247e7311cf2a1b83e0ccd27053/docs/_sources/_static/custom.css#L31-L33 https://github.com/FCP-INDI/fcp-indi.github.io/blob/0a067870948763247e7311cf2a1b83e0ccd27053/docs/_sources/_static/custom.css#L48-L53

Some of the class names (e.g., `bibtex` and `bibtex label` seem to be deprecated from some update of [`sphinxcontrib-bibtex`](https://sphinxcontrib-bibtex.readthedocs.io) though; I don't see any of those classes in the built docs after [v1.8.3](https://fcp-indi.github.io/docs/v1.8.3/user/tse#id1)

Reply:
Tutorial looks fine:
![tutorial screenshot](https://user-images.githubusercontent.com/5974438/217306284-16fb633f-cc8b-4a29-85d4-f6f97a72bb01.png)